rocks-bin/src/check.rs

```diff
@@ -0,0 +1,38 @@
+use eyre::{OptionExt, Result};
+use rocks_lib::{
+    build::BuildBehaviour,
+    config::Config,
+    lockfile::PinnedState::Pinned,
+    manifest::Manifest,
+    operations::{self, install},
+    progress::MultiProgress,
+    project::Project,
+};
+
+pub async fn check(config: Config) -> Result<()> {
+    let project = Project::current()?.ok_or_eyre("Not in a project!")?;
+
+    let src_dir = project.src_dir().ok_or_eyre(
+        "No source directory found! Expected a `src/` or `lua/` directory at the project root.",
+    )?;
```

Comment:
question: why not the project root?
People might want to check `spec` or `test` files too.

Reply:
That's a fair point, I haven't thought of that. Let's do just the root. Should I throw out the `src_dir()` function then if it's not used?

Reply:
> Should I throw out the src_dir() function then if it's not used?

Yeah, let's :+1:

Reply:
Oh, found a problem. Just setting the project root will also search `.rocks` hahah. We want to limit it somehow. There should be an `--exclude` flag or whatever.